packages/ffe-grid-react/src/Grid.md

```diff
@@ -91,6 +95,11 @@ const backgroundColors = [
     'grey-cloud',
     'purple-magenta',
     'sand',
+    'sand',
```

Comment:
`'sand'` seems to have been repeated in two lines.

Reply:
It's because there's a lot of it :D

Reply:
> It's coarse and rough and irritating and it gets everywhere.